examples/Bickley_jet_shallow_water.jl

```diff
@@ -0,0 +1,258 @@
+# # Shallow Water Example: an unstable Bickley jet 
+#
+# This example shows how to use `Oceananigans.ShallowWwaterModel` to simulate
+# the evolution of an unstable, geostrophically balanced, Bickley jet.
+# The model solves the governing equations for the shallow water model in
+# conservation form.  The geometry is that of a periodic channel
+# in the ``x``-direction with a flat bottom and a free-surface.  The initial
+# conditions are that of a Bickley jet with small amplitude perturbations.
+# The interested reader can see ["The nonlinear evolution of barotropically unstable jets," J. Phys. Oceanogr. (2003)](https://doi.org/10.1175/1520-0485(2003)033<2173:TNEOBU>2.0.CO;2)
+# for more details on this specific problem. 
+#
+# Unlike the other models, the fields that are simulated are the mass transports, 
+# ``uh`` and ``vh`` in the ``x`` and ``y`` directions, respectively,
+# and the height ``h``.  Note that ``u`` and ``v`` are the velocities, which 
+# can easily be computed when needed.
+#
+# ## Install dependencies
+#
+# First we make sure that we have all of the packages that are required to
+# run the simulation.
+# ```julia
+# using Pkg
+# pkg"add Oceananigans, NCDatasets, Plots, Printf, LinearAlgebra, Polynomials"
+# ```
+#
+# FJP: 1] Change the title in sidebar to be something else.
+#
+# FJP: 2] Need to reduce these lines here as this is way too much!
+
+using Oceananigans.Architectures: CPU, architecture
+using Oceananigans.Models: ShallowWaterModel
+using Oceananigans.Grids
+using Oceananigans.Grids: Periodic, Bounded, RegularRectilinearGrid
+using Oceananigans.Grids: xnodes, ynodes, interior
+using Oceananigans.Simulations: Simulation, set!, run!
+using Oceananigans.Coriolis: FPlane
+using Oceananigans.Advection: WENO5
+using Oceananigans.OutputWriters: NetCDFOutputWriter, TimeInterval, IterationInterval
+using Oceananigans.Fields, Oceananigans.AbstractOperations
```

Comment:
I had the impression that #1406 was done precisely for this reason, i.e., so users don't have to add all these `using ...`. Am I wrong? @francispoulin are you sure all these are necessary?

Reply:
Thanks for pointing this out and I agree that we want to reduce this list very much.  I'll try this out and see if I can get it to run okay.

Reply:
I'm happy to say this can all be reduced down to two lines, as you said. Much better!

```
using Oceananigans
using Oceananigans.Models: ShallowWaterModel
```